earth_enterprise/src/installer/uninstall_fusion.sh

```diff
@@ -159,21 +160,23 @@ determine_os()
     local test_versionid=""
 
     if [ -f "$OS_RELEASE1" ] || [ -f "$OS_RELEASE2" ]; then
-		if [ -f "$OS_RELEASE1" ]; then
-	        test_os="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^NAME= | sed 's:name=::gI')"
-    	    test_versionid="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^VERSION_ID= | sed 's:version_id=::gI')"
-		else
-			test_os="$(cat $OS_RELEASE2 | sed 's:[0-9\.] *::g')"
-			test_versionid="$(cat $OS_RELEASE2 | sed 's:[^0-9\.]*::g')"
-		fi
+	if [ -f "$OS_RELEASE1" ]; then
```

Comment:
This might make things a little tricky, but I moved this function to a common.sh script when doing install_server.sh so they can be shared between the two install scripts. It's not merged yet, so no harm, but we'll need to remember to copy these changes into there. Git isn't smart enough to figure out that text has been moved from one file to another.